build/repo.targets

```diff
@@ -0,0 +1,45 @@
+<Project>
+  <PropertyGroup>
+    <PublishFeed>https://dotnet.myget.org/F/aspnetcoremodule/api/v2/package</PublishFeed>
+    <VerifyDependsOn>$(VerifyDependsOn);PublishPackage</VerifyDependsOn>
+  </PropertyGroup>
+
+  <Target Name="BuildNativeAssets" DependsOnTargets="GetToolsets" BeforeTargets="Compile" >
+    <ItemGroup>
+      <BuildConfigurations Include="/p:Configuration=Release /p:platform=Win32" />
+      <BuildConfigurations Include="/p:Configuration=Release /p:platform=x64" />
+    </ItemGroup>
+
+    <Error
+      Text="Could not find an installation of Visual Studio with the C++ development tools."
+      Condition="'$(VisualStudioMSBuildx86Path)' == ''" />
+      
+    <Exec Command="&quot;$(VisualStudioMSBuildx86Path)&quot; &quot;$(RepositoryRoot)src\AspNetCore\AspNetCore.vcxproj&quot; %(BuildConfigurations.Identity)"
+      Condition="'$(VisualStudioMSBuildx86Path)' != ''" />
+    <Exec Command="&quot;$(VisualStudioMSBuildx86Path)&quot; &quot;$(RepositoryRoot)src\RequestHandler\RequestHandler.vcxproj&quot; %(BuildConfigurations.Identity)"
+      Condition="'$(VisualStudioMSBuildx86Path)' != ''" />
+  </Target>
+
+  <Target Name="PackageProjects">
+    <PropertyGroup>
+      <PackageVersion>1.0.0-pre-$(BuildNumber)</PackageVersion>
```

Comment:
You can remove this too. It is already be set in version.props.

https://github.com/aspnet/IISIntegration/blob/d2fba28272759f9c59ca9e08fd9d4373576a919d/version.props#L5-L6